refatoracaoCalculoImpostoDeRenda/src/Aliquota.java

```diff
@@ -0,0 +1,36 @@
+
+public abstract class Aliquota {
+    
+    private String nome;
+    private double rendaBruta;
+
+    
+    public String getNome() {
+        return nome;
+    }
+    
+    public void setNome(String nome) {
+        this.nome = nome;
+    }
+
+    
+    public double getRendaBruta() {
+        return rendaBruta;
+    }
+
+   
+    public void setRendaBruta(double rendaBruta) {
+        this.rendaBruta = rendaBruta;
+    }
+    
+    public abstract double calcularImposto();
+
+    @Override
+    public String toString() {

```

Comment:
Não entendi, cade o calculo por tipo?

`return(salario*_AliquotaImpostoPessoaJuridica/100);
           this.getCpf
           return(salario*_AliquotaPessoaFisica/100);`

Reply:
os cálculos estão separados em pessoa física :https://github.com/FabricioRamosCoelho/Desafios-OOP/edit/feature/refatoracaoCalculoImpostoDeRenda/refatoracaoCalculoImpostoDeRenda/src/PessoaFisica.java?pr=%2FFabricioRamosCoelho%2FDesafios-OOP%2Fpull%2F1 e em pessoa jurídica: https://github.com/FabricioRamosCoelho/Desafios-OOP/blob/feature/refatoracaoCalculoImpostoDeRenda/refatoracaoCalculoImpostoDeRenda/src/PessoaJuridica.java